app/assets/stylesheets/admin/calendar/_calendar.scss

```diff
@@ -79,6 +79,12 @@
     }
   }
 
+  .cal-step {
+    &.draft {
+      @extend .draft;
+    }
```

Comment:
I'm not sure that you need to use @extend here or below. If you define a global class, it should Already apply to any Element with that class, right? Try it without?

Reply:
That makes sense. I overcomplicated things I think.